frontend/templates/views/partials/analytics.j2

```diff
@@ -9,6 +9,20 @@
         }
       }
     }
+{% if '/courses/' in doc.pod_path and doc.title == 'Teachers' %}
```

Comment:
Will this work for translated versions? Why not only use the path?

Reply:
Sebastian was a little bit quicker 😁 Check for `doc.pod_path == '/content/amp-dev/documentation/courses/teachers.md'` instead.